bitnami/etcd/templates/snapshot-pvc.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.disasterRecovery.enabled -}}
```

Comment:
It would also be interesting to allow a user to specify an already existing PVC for snapshotting